base_layer/wallet/src/base_node_service/monitor.rs

```diff
@@ -126,6 +127,19 @@ where TWalletConnectivity: WalletConnectivityInterface
 
                     let is_synced = tip_info.is_synced;
                     let best_block_height = chain_metadata.best_block_height();
+                    trace!(
+                        target: LOG_TARGET,
+                        "Base node Tip: {} ({}) Latency: {} ms",
+                        best_block_height,
+                        if is_synced { "Synced" } else { "Syncing..." },
+                        latency.as_millis()
+                    );
+
+                    if best_block_height == last_checked_height {
```

Comment:
Maybe rather check best block hash as a new block can come in at the same height

Reply:
yeah, cahnged it to hash